host/xtest/stats.c

```diff
@@ -47,11 +47,11 @@ struct ta_dump_stats {
 	struct malloc_stats heap;
 };
 
-static const char *stats_progname = "xtest --stats";
+static const char *stats_option = "--stats";
 
 static int usage(void)
 {
-	fprintf(stderr, "Usage: %s [OPTION]\n", stats_progname);
+	fprintf(stderr, "Usage: %s %s [OPTION]\n", xtest_argv_0, stats_option);
```

Comment:
I'd rather have this function invoked as `usage(argv[0]);` by the various `stat_*()` functions because all other "applets" work like that.

Reply:
OK, i missed that. Thanks.

Reply:
see https://github.com/OP-TEE/optee_test/pull/672